### DISCUSSION TITLE
Compiling the code

### DISCUSSION THREAD

**Discussion by Jaaa67** [danielnilsson9/bbs-fw#237] [Q&A]
Hi, first of all thanks for the grate firmware, my bbsdh feels so much better now.
I would really like to tinker with the max temp value in the code. But I have no idea how I could convert it into a binary file.

**Comment by aalm** [comment]
Beside the obvious: https://github.com/danielnilsson9/bbs-fw/wiki/Build-from-Source
this can be be compiled on ie. a rpi4 w/linux. iirc. everything necessary can be installed w/apt.
There's a pull request with necessary changes to the makefile for that.
